go/mysql/binlog_event_json_test.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2019 The Vitess Authors.
+Copyright 2020 The Vitess Authors.
```

Comment:
we should _not_ update the copyright date on existing files.

Reply:
Had changed it because there were a large number of changes (>150 in ~175 lines). Changed it back.

Reply:
I think what we can do for substantial new changes is change it to `2019,2020` but I doubt it's worth it.